frontend/src/app/pages/exercises/exercise/shared/hospital-editor/hospital-editor-modal/hospital-editor-modal.component.html

```diff
@@ -74,6 +74,7 @@ <h4 class="modal-title">
                         (click)="removeHospital(hospital.id)"
                         class="btn btn-outline-danger float-end"
                         data-cy="hospitalDeleteButton"
+                        *ngIf="hospital.id !== catchAllHospitalId"
```

Comment:
Could you please disable the button instead of hiding it and proovide an explanation why it is disabled (e.g. with a tooltip)? Otherwise, users might be confused why they can't delete this hospital, especially if they might have renamed it, so they don't know its the default hospital.

Reply:
Done